styles/app.css

```diff
@@ -1,10 +1,133 @@
+/*round image, max width container, max width margin,*/
+
 body {
-  margin: 0
+  margin: 0;
   font-family: sans-serif;
 }
 
+header {
+  background-color: tomato;
+  padding: 1rem;
+}
+
+a {
+  font-size: 2rem;
+  text-decoration: none;
+
+  color: antiquewhite;
+}
+
 ul {
   padding-left: 0;
   list-style: none;
+  margin: 0;
+}
+
+img {}
+
+
+* {
+  box-sizing: border-box;
+}
+
+.nav-right {
+  float: right;
```

Comment:
Treat this list like you would a grid since the items within have margin left and right.

Use the negative margin trick to make up for the margin on `.nav-right a` that will make this edge line up better.

<img width="268" alt="screen shot 2017-02-01 at 9 10 13 pm" src="https://cloud.githubusercontent.com/assets/2532004/22535772/d8e22f54-e8c2-11e6-9449-1f61e6b17ce9.png">